### DISCUSSION TITLE
Community feedback on ROADMAP.md

### DISCUSSION THREAD

**Discussion by evgenyka** [aws/aws-cdk#29154] [General]
Evergreen discussion on [CDK public Roadmap](https://github.com/aws/aws-cdk/blob/main/ROADMAP.md).
We will update the roadmap monthly. Please add your feedback, comments, and suggestions!

**Comment by gshpychka** [comment]
Would really like to know what the current vision around autogenerated L2s is. The text mentioning it was removed from the roadmap without comment, could you shed some light on it? Here's my question on the original commit that removed it: https://github.com/aws/aws-cdk/commit/2988247436d3e032d48cc864e1157c8fa887112d#r143210346

Also left a comment on the RFC itself: https://github.com/aws/aws-cdk-rfcs/issues/611#issuecomment-2162801826

P.S.
> We will update the roadmap monthly. 

This conflicts with what `ROADMAP.md` [says](https://github.com/aws/aws-cdk/blob/775c58c9b73b877500b739fb1a1e7c21216a834c/ROADMAP.md?plain=1#L5):

`We update this doc on a quarterly basis to reflect any changing priorities.`
